src/views/customer/CustomerListView/Results.tsx

```diff
@@ -26,12 +27,47 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const Results: React.FC<any> = ({ className, customers, ...rest }) => {
+interface ITodo {
+  id: string;
+  text: string;
+  complete: boolean;
+}
+interface QueryResponse {
+  todos: ITodo[];
+}
+
+const SEARCH = gql`
+  query Search($search: String!) {
+    customers(where: {name: {_ilike: $search}}, order_by: {name: asc}) {
+      id
+      email
+      name
+      phone
+      address
+      avatarUrl
+      createdAt
+      updatedAt
+    }
+  }
+`;
+
+const Results: React.FC<any> = ({ className ,inputSearch, ...rest }) => {
```

Comment:
keep the original inputs
we need simple components (display only)
top level components/pages should do the queries/mutations